docs/Reference/Processors/AzureDevOpsPipelineProcessor.md

```diff
@@ -0,0 +1,56 @@
+## Processors: AzureDevOpsPipelineProcessor
+
+>**_This documentation is for a preview version of the Azure DevOps Migration Tools._ If you are not using the preview version then please head over to the main [documentation](https://nkdagility.github.io/azure-devops-migration-tools).**
+
+[Overview](.././index.md) > [Reference](../index.md) > [Processors](./index.md) > **AzureDevOpsPipelineProcessor**
+
+missng XML code comments
```

Comment:
Can you add a summary to the class. Thats where this tag comes from

Reply:
Done!